config/vufind/author-classification.ini

```diff
@@ -190,14 +200,15 @@ fmk = "Filmmaker"
 fmo = "Former owner"
 fmp = "Film producer"
 fnd = "Funder"
+fon = "Founder"
 fpy = "First party"
 frg = "Forger"
+gdv = "Game developer"
 gis = "Geographic information specialist"
 grt = "Graphic technician"
 his = "Host institution"
 hnr = "Honoree"
 hst = "Host"
-Ill = "Illustrator"
```

Comment:
If we remove this here, I wonder if we should add it to languages/CreatorRoles/aliases.ini as `Ill = ill` (and also remove it from all of the language files with `php $VUFIND_HOME/public/index.php language delete CreatorRoles::Ill`).

Reply:
Ah! I didn't know about that alias file. I pushed a commit to add a line.

How are translations handled in general? Do I need to do something other than add the lines to `en.ini`?

Reply:
Adding lines to `en.ini` is enough -- Lokalise will find the changes in GitHub and then when translators work on other languages, they'll be prompted to fill in missing translations.

Reply:
But as noted earlier, in addition to adding the alias, you should also delete the duplicate definition from all of the language files using the tool I mentioned. If that's not working right, maybe I mistyped something; let me know if you need me to dig deeper!

Reply:
Done! Thanks for your patience.